widget/entry.go

```diff
@@ -1067,8 +1068,11 @@ func (r *entryRenderer) Destroy() {
 }
 
 func (r *entryRenderer) Layout(size fyne.Size) {
-	r.line.Resize(fyne.NewSize(size.Width, theme.Padding()))
-	r.line.Move(fyne.NewPos(0, size.Height-theme.Padding()))
+	r.line.Resize(fyne.NewSize(size.Width, theme.EntryUnderlineSize()))
+	r.line.Move(fyne.NewPos(0, size.Height-theme.EntryUnderlineSize()))
+	// Substracting "r.box.StrokeWidth/2" in order to be ready to support OutlineBorder later in future.
```

Comment:
:)
I'm happy with this approach - but should the line perhaps come in on the left/right in the same manner?

Reply:
You mean substracting "r.box.StrokeWidth/2" in "line" too, right?

Reply:
I imagined this would be required for the line and background to align, yes. Just on the x/width dimension.

Reply:
Ok, I will do that change